packages/admin-ui-docs/docs/guidelines/components/alert.mdx

```diff
@@ -175,6 +175,58 @@ function Example() {
 
   </Panel>
   <Panel id="designReference">
+
+## Documentation
```

Comment:
This heading can be removed for all components since the documentation site only has documentation. I would then turn all the headings below that are currently in bold into H2s.